examples/template/template.js

```diff
@@ -63,6 +65,16 @@ const traceTemplates = [
             {service: "auth-service", name: "authenticate", attributes: {"http.status_code": 403}},
         ]
     },
+    {
+        defaults: traceDefaults,
+        spans: [
+            {service: "shop-backend", attributes: {"http.status_code": 403}},
+            {service: "shop-backend", name: "authenticate", attributes: {"http.request.header.accept": ["application/json"]}},
+            {service: "auth-service", name: "authenticate", attributes: {"http.status_code": 403}},
+            {service: "cart-service", name: "checkout", randomEvents: {exceptionCount: 1, count: 2, randomAttributes: {count: 5, cardinality: 2}}},
```

Comment:
When I test this together with the changes in PR grafana/tempo#3163 the events look a bit odd. 

* The timestamp is a negative value. In the trace data returned by tempo the timestamp is zero.
* Instead of an event `name` with the value `"exception"` there is a field `message` with the value `{"Type": "STRING", "Value": "exception"}`

TBH I'm not sure about the origin of those problems

![image](https://github.com/grafana/xk6-client-tracing/assets/2076136/ba73527c-c510-4e84-97d9-bc3ac5232f00)

Reply:
oh weird, I am using `grafana/grafana:main` and I do see `message` instead of `name` but it just shows "exception" not a struct

tempo:latest 
<img width="1038" alt="Screenshot 2024-01-16 at 1 06 32 PM" src="https://github.com/grafana/xk6-client-tracing/assets/94262131/350e25f2-11fd-467d-8498-eff16cc6e2d2">
<img width="1056" alt="Screenshot 2024-01-16 at 1 06 40 PM" src="https://github.com/grafana/xk6-client-tracing/assets/94262131/601b1497-a4c1-4625-b1e3-7032ced6ee49">
![Screenshot 2024-01-16 at 1 07 21 PM](https://github.com/grafana/xk6-client-tracing/assets/94262131/1e5d36e9-8795-4c16-87cf-7950e626af24)


tempo:pr3163
![Screenshot 2024-01-16 at 1 10 55 PM](https://github.com/grafana/xk6-client-tracing/assets/94262131/dc1db11b-0ff5-4ce1-8ec9-956af8876e17)
<img width="1035" alt="Screenshot 2024-01-16 at 1 11 30 PM" src="https://github.com/grafana/xk6-client-tracing/assets/94262131/c0182449-489b-4aa0-ac33-2b6d1e1e710f">
<img width="1026" alt="Screenshot 2024-01-16 at 1 11 39 PM" src="https://github.com/grafana/xk6-client-tracing/assets/94262131/e27db884-ad64-49ab-a1e9-a498bd3410e0">